app/viewmodels/checkAnswers/amends/AmendDirectExportPlasticPackagingSummary.scala

```diff
@@ -16,32 +16,32 @@
 
 package viewmodels.checkAnswers.amends
 
+import cacheables.ReturnDisplayApiCacheable
 import models.UserAnswers
+import models.amends.AmendSummaryRow
+import models.returns.ReturnDisplayApi
 import pages.amends.AmendDirectExportPlasticPackagingPage
 import play.api.i18n.Messages
-import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
-import viewmodels.checkAnswers.SummaryViewModel
-import viewmodels.govuk.all.FluentActionItem
-import viewmodels.govuk.summarylist.{ActionItemViewModel, SummaryListRowViewModel, ValueViewModel}
-import viewmodels.implicits._
-
-object AmendDirectExportPlasticPackagingSummary extends SummaryViewModel {
-
-  override def row(answers: UserAnswers)(implicit messages: Messages): Option[SummaryListRow] =
-    answers.get(AmendDirectExportPlasticPackagingPage).map {
-      answer =>
-        SummaryListRowViewModel(key = "amendDirectExportPlasticPackaging.checkYourAnswersLabel",
-          value = ValueViewModel(answer.toString),
-          actions = Seq(
-            ActionItemViewModel(
-              "site.change",
-              controllers.amends.routes.AmendDirectExportPlasticPackagingController.onPageLoad().url
-            )
-              .withVisuallyHiddenText(
-                messages("amendDirectExportPlasticPackaging.change.hidden")
-              )
-          )
-        )
-    }
+
+object AmendDirectExportPlasticPackagingSummary {
+
+  def apply(answers: UserAnswers)(implicit messages: Messages): AmendSummaryRow = {
+
+    val returnDisplayApi: ReturnDisplayApi = answers.get(ReturnDisplayApiCacheable).getOrElse(
+      throw new IllegalArgumentException("Must have a return display API to do an amend")
+    )
+
+    val maybeAnswer: Option[Int] = answers.get(AmendDirectExportPlasticPackagingPage)
```

Comment:
Can drop this line now and just do 

**_answers.get(AmendDirectExportPlasticPackagingPage).map(_.toString)** for 'amended'

Reply:
Same for others. 

@C-Thomp  - want to make this change?

Reply:
Yep on it